examples/calendar.rs

```diff
@@ -16,7 +16,7 @@ fn main() -> Result<(), Box<dyn Error>> {
     let mut terminal = Terminal::new(backend)?;
 
     loop {
-        let _ = terminal.draw(|f| draw(f));
+        let _ = terminal.draw(draw);
```

Comment:
Clippy dictated this change, which makes sense as the terminal method is concretely parameterized while the app method was generic and couldn't be called directly. Now it can.